doc/builders/images/appimagetools.section.md

```diff
@@ -35,7 +35,7 @@ appimageTools.wrapType2 { # or wrapType1
   name = "patchwork";
   src = fetchurl {
     url = "https://github.com/ssbc/patchwork/releases/download/v3.11.4/Patchwork-3.11.4-linux-x86_64.AppImage";
-    sha256 = "1blsprpkvm0ws9b96gb36f0rbf8f5jgmw4x6dsb1kswr4ysf591s";
+    hash = "sha256-1blsprpkvm0ws9b96gb36f0rbf8f5jgmw4x6dsb1kswr4ysf591s=";
```

Comment:
Shouldn't the sri hash be base64?

(Here and elsewhere)

Reply:
yeah, I may have the wrong syntax; some of this was guesswork on my part.

anything hex is going to be correct, as i manually regenerated those, but you are saying this is base32 currently but needs to be base64?

Reply:
You could do this

```
$ nix hash to-sri --type sha256 1blsprpkvm0ws9b96gb36f0rbf8f5jgmw4x6dsb1kswr4ysf591s
sha256-OqTitCeZ6xmWbqYTXp8sDrmVgTNjPZNW0hzUPW++mq4=
```

Although I suspect these hashes to be bogus anyway. However, one may reasonably expect an SRI hash to contain such things as upper case letters. A lack of those may again confuse people.

Reply:
yeah, I have some bandwidth and am happy to try and validate their real versions, so people can copy paste and stuff "just works"

Reply:
alright, everything should be fixed; let me know if that is better